scripts/cmux-workspace-handoff.py

```
@@ -0,0 +1,590 @@
+            primary = bundle / bundle_paths[0]
+            target = home / "projects" / target_slug / f"{descriptor['session_id']}.jsonl"
+            todo_targets = [home / "todos" / (bundle / rel_bundle).name for rel_bundle in bundle_paths[1:]]
+            ensure_no_overwrite(target)
+            for todo_target in todo_targets:
+                ensure_no_overwrite(todo_target)
+            target.parent.mkdir(parents=True, exist_ok=True)
+            shutil.copy2(primary, target)
+            if args.claude_cwd_mode == "rewrite":
+                rewrite_jsonl(target, old_cwd, str(dest))
+            placements.append(str(target))
+            for rel_bundle, todo_target in zip(bundle_paths[1:], todo_targets):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)